crt_portal/cts_forms/migrations/0161_constant_writer_form_letter_update.py

```diff
@@ -0,0 +1,18 @@
+from django.db import migrations
```

Comment:
(Nothing you can do right now, but keep in mind these migrations will create special merge conflicts with those from your other PR)

If you wanted to tackle this in advance you could rebase it onto the other PR and make a stacked PR

Reply:
@okkays @kewitham let's discuss this before merging.